cirq/contrib/quantum_volume/quantum_volume.py

```diff
@@ -475,9 +476,14 @@ def calculate_quantum_volume(
                                 depth=depth,
                                 num_circuits=num_circuits,
                                 random_state=random_state)
-    device_graph = ccr.gridqubits_to_graph_device(device_or_qubits) if type(
-        device_or_qubits) is list else ccr.xmon_device_to_graph(
-            device_or_qubits)
+
+    # Get the device graph from the given qubits or device.
+    device_graph = None
```

Comment:
thing of interest: usually you don't have to include this. most linters are smart enough to notice that `device_graph` is initialized no matter what and all the scoping in python is so horribly leaky you don't need to declare it outside

Reply:
mypy was complaining that the variable device_graph was referenced before assignment. I wasn't able to find an existing issue for this online, maybe its WAI?

Reply:
ah interesting